moveit_py/src/moveit/moveit_core/robot_trajectory/robot_trajectory.h

```diff
@@ -55,6 +56,9 @@ robot_trajectory::RobotTrajectory
 set_robot_trajectory_msg(const std::shared_ptr<robot_trajectory::RobotTrajectory>& robot_trajectory,
                          const moveit::core::RobotState& robot_state, const moveit_msgs::msg::RobotTrajectory& msg);
 
+bool retimeTrajectory(std::shared_ptr<robot_trajectory::RobotTrajectory>& self, const std::string& algorithm,
```

Comment:
Please add documentation for the functions you're creating

```cpp
  /**
   * \brief ...
   * \param [in] ...
   * ...
   * \return
   */
```

Reply:
Added documentation for all functions in trajectory_tools.h